src/components/NotificationsDrawer/DrawerPanelContent.tsx

```diff
@@ -39,19 +56,170 @@ const EmptyNotifications = () => (
+      Mark visible as read
+    </DropdownItem>,
+    <DropdownItem key="unread all" onClick={() => onMarkAllAsUnread()} isDisabled={notifications.length === 0}>
+      Mark visible as unread
+    </DropdownItem>,
+    <Divider key="divider" />,
+    <DropdownItem key="event log">
+      <Flex>
+        <FlexItem>View event log</FlexItem>
+        <FlexItem align={{ default: 'alignRight' }}>
+          <Icon className="pf-v5-u-ml-auto">
+            <ExternalLinkAltIcon />
+          </Icon>
+        </FlexItem>
+      </Flex>
+    </DropdownItem>,
+    isOrgAdmin && (
+      <DropdownItem key="notification settings">
+        <Flex>
+          <FlexItem>Configure notification settings</FlexItem>
+          <FlexItem align={{ default: 'alignRight' }}>
+            <Icon className="pf-v5-u-ml-auto">
+              <ExternalLinkAltIcon />
+            </Icon>
+          </FlexItem>
+        </Flex>
+      </DropdownItem>
+    ),
+    <DropdownItem key="notification preferences">
+      <Flex>
+        <FlexItem>Manage my notification preferences</FlexItem>
+        <FlexItem align={{ default: 'alignRight' }}>
+          <Icon className="pf-v5-u-ml-auto">
+            <ExternalLinkAltIcon />
+          </Icon>
+        </FlexItem>
+      </Flex>
+    </DropdownItem>,
+  ];
+
+  const filterDropdownItems = () => {
+    return [
+      <DropdownGroup key="filter-label" label="Show notifications for...">
+        {filterConfig.map((source, index) => (
+          <DropdownItem key={index} onClick={() => onFilterSelect(source.value)}>
+            <Checkbox isChecked={activeFilters.includes(source.value)} id={index.toString()} className="pf-v5-u-mr-sm" />
+            {source.title}
+          </DropdownItem>
+        ))}
+        <Divider />
+        <DropdownItem key="reset-filters" onClick={() => setActiveFilters([])}>
+          <Button variant="link" isDisabled={activeFilters.length === 0} isInline>
+            Reset filters
+          </Button>
+        </DropdownItem>
+      </DropdownGroup>,
+    ];
+  };
+
+  const renderNotifications = () => {
+    if (notifications.length === 0) {
+      return <EmptyNotifications />;
+    }
+
+    const sortedNotifications = orderBy(
+      filteredNotifications?.length > 0 ? filteredNotifications : notifications,
+      ['read', 'created'],
+      ['asc', 'asc']
+    );
+
+    return sortedNotifications.map((notification, index) => <NotificationItem key={index} notification={notification} />);
+  };
+
   return (
     <NotificationDrawer ref={innerRef}>
-      <NotificationDrawerHeader onClose={() => dispatch(toggleNotificationsDrawer())}>
-        <Button variant="plain" aria-label="Filter">
-          <FilterIcon />
-        </Button>
-        <Button variant="plain" aria-label="Mark as read">
-          <EllipsisVIcon />
-        </Button>
+      <NotificationDrawerHeader onClose={() => onNotificationsDrawerClose()}>
```

Comment:
No need to create anonymous function
```suggestion
      <NotificationDrawerHeader onClose={onNotificationsDrawerClose}>
```